NOTES.md

```diff
@@ -1,3 +1,246 @@
+# Release 1.16.0 (Stand By Me)
+
+Released on 2015-01-29
+
+The highlights of this release are:
+* A new administration API
+* Changefeeds on complex queries
+* Numerous improvements and enhancements all over the place
+
+Read the [release blog post][1.16-blog] for more details.
+
+[1.16-blog]: http://rethinkdb.com/blog/1.16-release/
+
+## Compatbility ##
+
+### Backwards-compatible changes ###
+
+Data files from RethinkDB versions 1.13.0 onward will be automatically
+migrated to version 1.16.x. As with any major release, back up your data files
+before performing the upgrade. If you are upgrading from a release earlier
+than 1.13.0, follow the migration instructions before upgrading:
+
+http://rethinkdb.com/docs/migration/
+
+Secondary indexes now use a new format; old indexes will continue to work, but
+you should rebuild indexes after upgrading to 1.16.x. A warning about outdated
+indexes will be issued on startup.
+
+Indexes can be migrated to the new format with the `rethinkdb index-rebuild`
```

Comment:
Can we use reference-style links here as well? e.g.:

> Consult the [troubleshooting document][outdated-indexes] for more information.

[outdated-indexes]: http://rethinkdb.com/docs/troubleshooting#my-secondary-index-is-outdated